kobo/apps/openrosa/apps/logger/models/attachment.py

```diff
@@ -1,4 +1,3 @@
-# coding: utf-8
```

Comment:
Why was this removed?

Reply:
When I was adding the linter to CI, I came across [this rule](https://docs.astral.sh/ruff/rules/utf8-encoding-declaration/). I did not want to enforce it in the CI but I decided to remove them manually when I was touching a file.